src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -654,7 +654,7 @@ public Response consumerExistsBulk(Set<String> consumerUuids) {
     }
 
     @Override
-    public ConsumerDTO getConsumer(@Verify(Consumer.class) String uuid) {
+    public ConsumerDTO getConsumer(@Verify(Consumer.class) String uuid, String acceptLanguage) {
```

Comment:
I *really* don't want to see all of the various headers and parameters start showing up in methods like this. Do we have a better way of doing the OpenAPI stuff to avoid this?

Reply:
With the new generic Requests API (https://github.com/candlepin/candlepin/pull/3527), you can create generic requests and set the language header there, and revert the spec file change